app/services/diagnostics/move_inspector.rb

```diff
@@ -37,13 +37,13 @@ def generate
       ENDMOVEEVENTS
 
       if move.generic_events.any?
-        @output << "#{'EVENT'.ljust(30)}\t#{'TIMESTAMP'.ljust(27)}\t#{'NOTES'.ljust(30)}\tDETAILS\n"
+        @output << "#{'EVENT'.ljust(30)}\t#{'OCCURRED AT'.ljust(27)}\t#{'NOTES'.ljust(30)}\tDETAILS\n"
         move.generic_events.applied_order.each do |event| # NB use each to preserve sort order
           # NB only show event params if include_person_details==true, as they could contain personal details
-          @output << "#{event.type.ljust(30)}\t#{event.occurred_at.to_s.ljust(27)}\t#{include_person_details ? event.notes.to_s.truncate(30).ljust(30) : '-'.ljust(30)}\t#{include_person_details ? event.details : '-'}\n"
+          @output << "#{event.type.delete_prefix('GenericEvent::').ljust(30)}\t#{event.occurred_at.to_s.ljust(27)}\t#{include_person_details ? event.notes.to_s.truncate(30).ljust(30) : '-'.ljust(30)}\t#{include_person_details ? event.details : '-'}\n"
```

Comment:
I think you can use `event_type` here instead, which removes the prefix: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/models/generic_event.rb#L116

Reply:
thanks, didn't know about that 👍